configs.go

```diff
@@ -2,19 +2,28 @@ package tgbotapi
 
 import (
 	"encoding/json"
+	"fmt"
 	"io"
 	"net/url"
 	"strconv"
 )
 
-// Telegram constants
-const (
-	// APIEndpoint is the endpoint for all API methods,
-	// with formatting for Sprintf.
+// APIEndpoint is the endpoint for all API methods,
+// with formatting for Sprintf.
+var APIEndpoint string
+// FileEndpoint is the endpoint for downloading a file from Telegram.
+var FileEndpoint string
+
+func init() {
 	APIEndpoint = "https://api.telegram.org/bot%s/%s"
-	// FileEndpoint is the endpoint for downloading a file from Telegram.
 	FileEndpoint = "https://api.telegram.org/file/bot%s/%s"
-)
+}
+
+// SwitchAPIEndpointToLocal switch the API endpoint to local bot api server
+// with customize port
+func SwitchAPIEndpointToLocal(port int) {
+	APIEndpoint = fmt.Sprintf("http://127.0.0.1:%d", port) + "/bot%s/%s"
```

Comment:
I think better is to setup URL by user (not 127.0.0.1), e.g. in case when app & custom bot API server is not on the same machine

Reply:
you are right, thanks